woocommerce/woocommerce-android
# Dark Mode: Order Products Card

Partially fixes #1587 by implementing light/dark material styling on the order detail products list card. Note that this PR **only**  covers the order detail products list card: 

### Collapsed Version:

<img src="https://user-images.githubusercontent.com/5810477/74076555-d8a86e00-49d5-11ea-8882-80e11d4ce083.png" width="300"/>

### Expanded Version:

<img src="https://user-images.githubusercontent.com/5810477/74076552-d7774100-49d5-11ea-8661-e71c06a31ba2.png" width="300"/>


This card displays the list of products for an order in the following screens:
- Order Details View
- Order Product Details View
- Order Fulfillment View


## Screenshots
![order-product-card-processing](https://user-images.githubusercontent.com/5810477/74076447-5029cd80-49d5-11ea-9f0d-b7d1a429eca4.png)

![order-product-card-completed](https://user-images.githubusercontent.com/5810477/74076452-528c2780-49d5-11ea-8c5c-83785d5421c4.png)

![order-product-details](https://user-images.githubusercontent.com/5810477/74076454-55871800-49d5-11ea-9668-00d02bcac074.png)

## To Test
Run the following tests on a test devices running **API 23 & 29**, as well as in both **light** and **dark** mode:
1. Open an order with a status of `processing` and verify styles displayed appropriately
2. Click "Begin Fulfillment" button and verify product details displayed properly
3. Mark order as complete
4. On the order details screen, click the "DETAILS" button to open the product details. 
6. Verify product details displayed properly.
